packages/ui-voip/src/components/VoipPopup/views/VoipErrorView.tsx

```diff
@@ -54,6 +55,8 @@ const VoipErrorView = ({ session, position }: VoipErrorViewProps) => {
 			</Footer>
 		</Container>
 	);
-};
+});
+
+VoipErrorView.displayName = 'VoipErrorView';
```

Comment:
Using a named function as the render function in `forwardRef()` does the same trick.